app/services/service_listeners/publishing_api_html_attachments.rb

```diff
@@ -18,8 +18,20 @@ def publish
     alias_method :unwithdraw, :publish
 
     def republish
-      update_publishing_api_content
-      unpublish_if_required
+      if Edition::PRE_PUBLICATION_STATES.include?(edition.state)
+        update_draft(update_type: "republish")
+      elsif edition.unpublishing && edition.withdrawn?
+        do_publish("republish")
+        discard_drafts(deleted_html_attachments)
+        withdraw
+      elsif edition.unpublishing
```

Comment:
Similarly we could use `edition.unpublished?` here, I think

Reply:
yup I can apply this